package.json

```diff
@@ -93,7 +93,7 @@
     "playwright": "https://gitpkg.now.sh/elastic/playwright/packages/playwright-core?1.32.3-recorder",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
-    "styled-components": "^5.3.7",
+    "styled-components": "^6.0.0-rc.1",
```

Comment:
is this worth doing right now given its a release candidate? Also why not just go to rc.3 if we want to proceed with the RC.

Reply:
I think I upgraded for `npm audit` reasons. Agreed though, will downgrade 👍

Reply:
Fixed c7782d57f90684101fb7fa16e4f98b510fad26f6